photon-api/src/integTest/scala/com/linkedin/photon/ml/model/GAMEModelTest.scala

```diff
@@ -87,19 +86,19 @@ class GAMEModelTest extends SparkTestUtils {
     val REModel2 = getRandomEffectModel(sc, 2)
 
     // case 1: fixed effect model only
-    val FEModelOnly = GAMEModel(LOGISTIC_REGRESSION, (FEModelName1, FEModel1), (FEModelName2, FEModel2))
+    val FEModelOnly = GAMEModel((FEModelName1, FEModel1), (FEModelName2, FEModel2))
     assertEquals(FEModel1, FEModelOnly.getModel(FEModelName1).get)
     assertEquals(FEModel2, FEModelOnly.getModel(FEModelName2).get)
     assertTrue(FEModelOnly.getModel(REModelName1).isEmpty)
 
     // case 2: random effect model only
-    val REModelOnly = GAMEModel(LOGISTIC_REGRESSION, (REModelName1, REModel1), (REModelName2, REModel2))
```

Comment:
We should add a in README somewhere that we use FE for fixed effect and RE for random effect. Would just be nice for the readers.